src/fe_image.cpp

```diff
@@ -237,8 +237,12 @@ FeTextureContainer::FeTextureContainer(
 	m_swf( NULL ),
 	m_movie_status( -1 ),
 	m_video_flags( VF_Normal ),
-	m_mipmap( false )
+	m_mipmap( false ),
+	m_active_texture( 0 )
 {
+	m_texture[0] = new sf::Texture;
+	m_texture[1] = new sf::Texture;
```

Comment:
I think you have some memory leakage here, I don't see m_texture[] getting deleted when the fetexturecontainer is destructed

Reply:
Thanks, I will fix it.